lib/virt_autotest/common.pm

```diff
@@ -339,4 +339,31 @@ if (get_var("REGRESSION", '') =~ /xen/) {
     %guests = ();
 }
 
+our %imports = ();
```

Comment:
The naming is not really descriptive - it took me until the end to realize what `imports` are.
Is it necessary to distinguish between guests and "imported guests"? Can't we add the windows-vms to `%guests`?

Otherwise I suggest to add a note to explain what they are, something like

    # imports are virtual machines, running on a hypervisor that we don't test ("imported virtual machines")

I think that would improve the readability of the procedures

Reply:
I'd say:
```
# imports are virtual machines that we don't install but just import. We test those separately.
```
is it OK?